addons/test_mail/tests/test_mail_channel.py

```diff
@@ -94,16 +94,19 @@ def test_access_rights_followers_portal(self):
         # Do: Chell is added into Pigs members and browse it -> ok for messages, ko for partners (no read permission)
         self.group_private.write({'channel_partner_ids': [(4, self.user_portal.partner_id.id)]})
         chell_pigs = self.group_private.with_user(self.user_portal)
-        trigger_read = chell_pigs.name
+        _ = chell_pigs.name
```

Comment:
is `_` a special magic stuff ???
because I would say that naming is quite missing..